hermes/util.py

```diff
@@ -2,71 +2,46 @@
 Project-wide utilities.
 """
 
-import collections
 import logging
-import tornado
-from jinja2 import Environment, PackageLoader
+import requests
 
 from .settings import settings
-from .version import __version__
 
 
 log = logging.getLogger(__name__)
 
-_TRUTHY = set([
-    "true", "yes", "1", ""
-])
 
+class SlackHelper(object):
```

Comment:
Why did you got with the decision make a class here over a standard function? You have a single class method and you never actually use cls being passed in.